spec/recurly/api/errors_spec.rb

```diff
@@ -33,15 +33,15 @@
       end
 
       describe "when response body is empty string" do
-        before { response.expect :body, '' }
+        before { 2.times { response.expect :body, '' } }
```

Comment:
I am not a MiniTest expert, however as best as I can tell, old versions of minitest would simply use the same response for multiple calls to the same mock. It seem that in later versions you need to specify the response for each expected call in the expected order (http://stackoverflow.com/questions/26806274/multiple-invocations-of-same-method-on-a-mock-with-minitest). In this specific case the call issues at https://github.com/recurly/recurly-client-ruby/blob/master/lib/recurly/api/errors.rb#L49 will result in two calls to .body the first call could be services and the second would result in an error because the expectation did not know how to respond.